supervision/detection/track.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from supervision.detection.core import Detections
+from supervision.geometry.core import Position
+
+
+class TrackStorage:
+    """
+    Trace the object trajectory with Detections with tracker id
+    """
+
+    def __init__(
+        self,
+        position: Position = Position.CENTER,
+        max_length: int = 10,
+    ):
+        """
+        Initialize the track storage to store tracked detections from tracker
+        Args:
+            position [sv.Position]: Trace position whethere mid-point or bottom center point
+            max_length [int]: Length of previous detections to annotate
+            Detections objects are stored as numpy ndarray with #counter, x, y, class, track_id
+
+        Example:
+            ```python
+            >>> import supervision as sv
+            >>> track_storage = sv.TrackStorage()
+            >>> for frame in sv.get_video_frames_generator(source_path='source_video.mp4'):
+            >>>     detections = sv.Detections(...)
+            >>>     tracked_objects = tracker(...)
+            >>>     tracked_detections = sv.Detections(tracked_objects)
+            >>>     track_storage.update(tracked_detections)
+        """
+        self.position = position
+        self.max_length = max_length
+        self.frame_counter = 0
+        self.storage = np.zeros((0, 5))
```

Comment:
Curious why you decided to use `np.ndarray` as a storage engine.

Reply:
> Curious why you decided to use `np.ndarray` as a storage engine.

Hi @SkalskiP, I am trying to rewrite `TrackerStorage` in the format of `Detections`. I will let you know once I am finished.

Reply:
@SkalskiP I am done with consistent format for `TrackerStorage`. You can review it now.